easybuild/framework/easyblock.py

```diff
@@ -127,9 +128,26 @@
 # Directory name in which to store reproducibility files
 REPROD = 'reprod'
 
+CHECKSUMS_JSON = 'checksums.json'
+
 _log = fancylogger.getLogger('easyblock')
 
 
+def update_progress_bar_on_return(func):
```

Comment:
@smoors Wouldn't it be better to make this a private decorator (rename to start with `_`), and to also mention `obtain_file` in the name, since it's quite specific to that method?

Reply:
fixed in 33e0ded14